doc/nrf/ug_zigbee_commissioning.rst

```diff
@@ -0,0 +1,208 @@
+
+.. figure:: /images/zigbee_commissioning_steering.svg
+   :alt: Zigbee Network Steering procedure
+
+   Zigbee Network Steering procedure
+
+How the node acts in this mode depends on whether the node is in a Zigbee network or not:
+
+* If the node is already a member of a Zigbee network, it opens the network and then closes it after the specified period of time (180 seconds by default).
+  The opening of the network is triggered when any of the Router or Coordinator nodes broadcasts the permit join request.
+* If the node is not a member of a Zigbee network, it scans channels for an open network to associate with.
+  Once it finds a suitable network, it attempts to join it and get the credentials.
+  If the node finds more than one available network, it will attempt to join them individually.
+
+If the node does not find a network to join and it has either the Coordinator or the Router role, it can attempt to form a new network (see `Network Formation`_).
+If the node has the End Device role, its commissioning procedure stops here.
+
+If the node finds a network to join, it can continue to the `Finding and Binding`_ procedure if it is requested with the appropriate bit mask value.
+
+The new device must be authenticated and authorized to become part of the network.
+For this purpose, it needs credentials, which usually take form of a network key.
+For `Centralized security network`_, the authentication and authorization also involves establishing a unique Trust Center Link Key (TCLK).
+
+Network Steering in the |NCS|
+   In the ZBOSS stack, Network Steering is started using the `ZB_BDB_NETWORK_STEERING`_ bit mask.
+
+   In the |NCS|, Network Steering for a node on a network can be started using the following additional options:
+
+   * When running the :ref:`Zigbee Coordinator sample <zigbee_network_coordinator_sample>`, you can press the associated button.
+   * If you are using the :ref:`lib_zigbee_shell` library in your application, you can use the ``bdb start`` command.
+     This command always starts Network Steering for the Router role.
+     For the Coordinator role, this command only works when the device is in the network; when not in the network, ``bdb start`` starts Network Formation.
+
+.. _zigbee_commissioning_modes_nf:
+
+Network Formation
+=================
+
+The Network Formation mode is available only to Coordinators and Routers, because End Devices are not capable of forming a Zigbee network.
+
+.. figure:: /images/zigbee_commissioning_formation.svg
+   :alt: Zigbee Network Formation procedure
+
+   Zigbee Network Formation procedure
+
+In this mode, the node has not found a network to join during the Network Steering mode, and so it attempts to form a new network.
+Depending on the node role:
+
+* A Coordinator forms a centralized security network.
+* If configured for forming a nework, a Router forms a distributed security network.
```

Comment:
```suggestion
* If configured for forming a network, a Router forms a distributed security network.
```

Reply:
Out of curiosity, `configured for` or `configured to`? Is the other one acceptable?

Reply:
In this context, "configured for" equals "meeting requirements for", so it's ok.